data_architect_misc/data_transformer/transform_functions/common_comp_harm_transform_functions.py

```diff
@@ -332,6 +357,7 @@ def add_HARMONIZED_COUNTRY_column_using_fixed_str_value(
                  comp_harm_constants.COUNTRY_COLUMN,
                  fixed_str_value)
 
+
```

Comment:
Remove this extra line. PEP8 (python's recommended formatting guidelines) prefers that we leave only one extra line between functions/methods inside a class. Pycharm normally takes care of that. Maybe you are using VSCode, so the linter you're using there might be not configured for Python.